recipes/jasper/all/patches/3.0.6-0002-find-libjpeg.patch

```diff
@@ -0,0 +1,18 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index a0d253d..19518af 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -680,12 +680,7 @@ if(JAS_ENABLE_LIBJPEG)
+ 		# (e.g., stdio.h and stdint.h).  So, we cannot reliably use
+ 		# check_include_file here.
+ 		jas_get_includes_from_targets(CMAKE_REQUIRED_INCLUDES JPEG::JPEG)
+-		check_c_source_compiles("
+-			#include <stdio.h>
+-			#include <stdint.h>
+-			#include <jpeglib.h>
+-			int main() {}
+-		" JAS_HAVE_JPEGLIB_H)
++		set(JAS_HAVE_JPEGLIB_H 1)
```

Comment:
same issue than https://github.com/conan-io/conan/issues/12180?

Reply:
I think so, indeed it does not work with `check_c_source_compiles`